compiler/rustc_trait_selection/src/traits/specialize/specialization_graph.rs

```diff
@@ -327,8 +331,12 @@ impl GraphExt for Graph {
         loop {
             use self::Inserted::*;
 
-            let insert_result =
-                self.children.entry(parent).or_default().insert(tcx, impl_def_id, simplified)?;
+            let insert_result = self.children.entry(parent).or_default().insert(
+                tcx,
+                impl_def_id,
+                simplified,
+                self.overlap_mode,
```

Comment:
... I was expecting this to be a parameter to the function, I think. Maybe that's annoying to do?